dist/profile/manifests/kubernetes/apply.pp

```diff
@@ -24,46 +24,56 @@
 #
 define profile::kubernetes::apply (
   String $resource = $title,
+  String $context = '',
```

Comment:
There is no documentation for what `context` is supposed to be for using this defined type. Please update the Puppet  Doc above.